modules/core/src/main/scala/org/tessellation/l0/DAGSnapshot.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.l0
+
+import org.tessellation.dag.types.DAGBlock
+import org.tessellation.schema.height.Height
+import org.tessellation.security.Hashed
+
+case class DAGSnapshot(height: Height, blocks: Set[Hashed[DAGBlock]])
```

Comment:
In 1.0 height was dynamically calculated based on `blocks`. We should probably do the same here. WDYT?

Reply:
I think it can be dynamic as well. Also, facilitators are missing (next facilitators)

Reply:
Yeah we need to figure out if there will be a rotation happening on L0 for DAG. Not sure if we need to add it right now or wait. What about lastSnapshot?

Reply:
We don't need to add reputation yet I think.

Reply:
Also I intended `Hashed` to be used internally, not sent over network as every node should regenerate hashes on it's own. So just like `DAGBlock` stores  `Set[Signed[Transaction]]` maybe the same case should be for the blocks? So not `Set[Hashed[DAGBlock]]` but `Set[Signed[DAGBlock]]`

Reply:
Other thing. Maybe we should keep the same principal as we do for transactions and blocks, so we have a complete reference to the previous snapshot? Given that it's like transactions `Height` and `Hash`, wdyt?

Reply:
We will need the same rotation in facilitators for L0 as for L1, at least in the final implementation.

Reply:
@TheMMaciek but `BlockStorage` doesn't have `Signed[DAGBlock]`. You would need to store there `Hashed[Signed[DAGBlock]]` or sth 🤔

Reply:
`Hashed` has the `Signed` inside of it, you can just extract it.